{"name": "test_divide[4-7-11]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0.5714285714285714 == 11\n  +0.5714285714285714\n  -11", "trace": "self = <test_calculator.TestCalculator object at 0x0000017CA0C71640>, a = 4, b = 7, exc = 11\n\n    @pytest.mark.parametrize('a,b, exc', [\n        (1, 2, 3),\n        (2, 4, 5),\n        (4, 7, 11)\n    ])\n    def test_divide(self, a, b, exc):\n        if b != 0:\n            result = self.c.divide(a, b)\n>           assert result == exc\nE           assert 0.5714285714285714 == 11\nE             +0.5714285714285714\nE             -11\n\ntest_calculator.py:55: AssertionError"}, "parameters": [{"name": "a", "value": "4"}, {"name": "b", "value": "7"}, {"name": "exc", "value": "11"}], "start": 1615726260898, "stop": 1615726260898, "uuid": "bfea576a-05e0-42b9-915f-22eeb0ffdbbe", "historyId": "a51be1583f54b1f8a67a7409d28b2f56", "testCaseId": "2c8d772870fd08f5fb143f47cbd63015", "fullName": "test_calculator.TestCalculator#test_divide", "labels": [{"name": "suite", "value": "test_calculator"}, {"name": "subSuite", "value": "TestCalculator"}, {"name": "host", "value": "FDD-PZSTWIEXCFJ"}, {"name": "thread", "value": "2028-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calculator"}]}